src/ripple/app/paths/Flow.cpp

```diff
@@ -139,6 +147,8 @@ flow (
 
         if (f.ter == tesSUCCESS)
             strandSB.emplace (std::move (*f.sandbox));
+        else
+            removableOffers = std::move (f.removableOffers);
```

Comment:
I'm seeing a fair amount of code duplication building up here.  Sometimes that can lead to maintenance issues in the future.  I'm thinking a generic lambda might help.  Consider this approach: https://github.com/scottschurr/rippled/commit/80cd0d5de08edea26d326c32e3c6a9c7164733a2.  Just a thought.  Use your best judgement.